src/components/Breadcrumb/BreadcrumbItem.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { StyledBreadcrumbItem } from "./BreadcrumbItem.styled";
+import styled from "styled-components";
+
+const StyledBreadcrumbElement = styled.li`
+    & + &::before {
+        content: '/';
+        padding: 8px
+    }
+    ${props=> props.active && {color:'#6c757d'}}
+`
```

Comment:
Tak jak wspominałem - można też mieć jeden plik `.styled` i do niego wrzucać komponenty stylowane - nawet jeśli jest to jakieś pierdółka ;)